addons/noactions/stringtable.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="TAC">
+    <Package name="No Actions">
+        <Key ID="STR_TAC_NoActions_DisplayName">
+            <English>No Actions</English>
+        </Key>
+        <!-- Eject  -->
+        <Key ID="STR_TAC_NoActions_Eject">
+            <English>Hide Eject action</English>
+            <German>Abspringen-Aktion entfernen</German>
+        </Key>
+        <Key ID="STR_TAC_NoActions_Eject_Desc">
+            <English>Hides the Eject entry from the action menu. Requires a game restart.</English>
```

Comment:
```suggestion
            <English>Hides the Eject entry from the action menu.</English>
```
No need for this anymore as CBA now has an icon and note for this - which you already use.

Reply:
Yes and no. CBA icon says that it requires a mission restart, this requires game restart.
It's consistent with ACE: https://github.com/acemod/ACE3/pull/6868/files#diff-ea946511ca5f9a27b1508f4da61c64f2R84 (guess from where he got the text from :D)

Also I should remove entries from "eject" action as it's in ACE already

Reply:
Ah ok!